opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/exclusionListResolver-test.ts

```diff
@@ -0,0 +1,161 @@
+import { describe, it, expect, beforeAll } from 'vitest';
+import gql from 'graphql-tag';
+import Upload from 'graphql-upload/Upload.mjs';
+import { downloadFile, streamConverter } from '../../../src/database/file-storage';
+import { fileToReadStream } from '../../../src/database/file-storage-helper';
+import { elLoadById } from '../../../src/database/engine';
+import { ADMIN_USER, testContext } from '../../utils/testQuery';
+import { queryAsAdminWithSuccess } from '../../utils/testQueryHelper';
+import { ExclusionListEntityTypes } from '../../../src/generated/graphql';
+
+const CREATE_CONTENT_MUTATION = gql`
+  mutation exclusionListContentAdd($input: ExclusionListContentAddInput!) {
+    exclusionListContentAdd(input: $input) {
+      id
+      file_id
+    }
+  }
+`;
+
+const CREATE_FILE_MUTATION = gql`
+  mutation exclusionListFileAdd($input: ExclusionListFileAddInput!) {
+    exclusionListFileAdd(input: $input) {
+      id
+      file_id
+    }
+  }
+`;
+
+const DELETE_MUTATION = gql`
+  mutation exclusionListDelete($id: ID!) {
+    exclusionListDelete(id: $id)
+  }
+`;
+
+type ExclusionListResponse = {
+  id: string | null;
+  file_id: string | null;
+};
+
+describe('Exclusion list resolver', () => {
+  let exclusionListResponse: ExclusionListResponse = { id: null, file_id: null };
+  let exclusionListFileResponse: ExclusionListResponse = { id: null, file_id: null };
+
+  describe('exclusionListValuesAdd', () => {
+    describe('If I create an exclusion with a content', () => {
+      beforeAll(async () => {
+        const result = await queryAsAdminWithSuccess({
+          query: CREATE_CONTENT_MUTATION,
+          variables: {
+            input: {
+              name: 'test_name',
+              description: 'test_description',
+              list_entity_types: [ExclusionListEntityTypes.DomainName],
+              content: 'test_content.fr'
+            }
+          }
+        });
+        if (result?.data?.exclusionListContentAdd) {
```

Comment:
why a 'if' there? we should know in this test whether or not result?.data?.exclusionListContentAdd is defined